File: Arcade/Intro/03-checkPalindrome.java
boolean checkPalindrome(String inputString) {
    int counter = 0;
    int counter_final = inputString.length() - 1;
    
    while (counter < counter_final){
        if (inputString.charAt(counter) == inputString.charAt(counter_final)){
            counter++;
            counter_final--;
        } else {
            return false;
        }
        
    }
    return true;
}

File: Arcade/Intro/03-checkPalindrome.py
def checkPalindrome(inputString):
    if len(inputString) == 1:
        return True
    idx_start = 0
    idx_end = len(inputString) - 1
    while (idx_start < idx_end):
        if inputString[idx_start] == inputString[idx_end]:
            idx_start += 1
            idx_end -= 1
        else:
            return False
    return True